### DISCUSSION TITLE
Unexpected force push of onboarding branch

### DISCUSSION THREAD

**Discussion by rarkins** [renovatebot/renovate#26332] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

None

### Describe the problem

This PR received an unexpected force push: https://github.com/import-js/.github/pull/2

It seems like Renovate thought the config changed and needed to update the PR:

```
DEBUG: Rebasing onboarding branch
{
  "previousConfigHash": "16b5d9aaa4c1b72cbe8c97df5985b530c94d3cb61c84931d45dcc539f5fbf9f7"
  "currentConfigHash": "e80b4e42a3043bc12fa0640db4bac392d2bf770acf841360d7c8ceeeac2ec1a9"
}
```

Previous run log (no rebase): https://gist.github.com/rarkins/b50fc1c548cbc361b863ae5e6a643a91
Second run log (where rebase happened): https://gist.github.com/rarkins/55ebb5b4da2af2446926f76bafb731c6

### Relevant debug logs

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
@RahulGautamSingh can you compare the configs from the two logs and work out why the hash changed and if it's intentional/desirable or if logic can be improved to prevent this in future?

**Reply by RahulGautamSingh** [reply]
It happened because in the two runs the onboarding config changed.
Previously
```
  "onboardingConfig": {
      "$schema": "https://docs.renovatebot.com/renovate-schema.json",
      "extends": [
        "config:base"
      ]
    },
```

Now
```
  "onboardingConfig": {
      "$schema": "https://docs.renovatebot.com/renovate-schema.json",
      "extends": [
        "config:recommended"
      ]
    },
```

**Reply by rarkins** [reply]
OK, that's a rare change and expected then
